object/check.go

```diff
@@ -42,9 +42,17 @@ func CheckUserSignup(application *Application, organization *Organization, usern
 	if application.IsSignupItemVisible("Username") {
```

Comment:
Modify to pure text:

![B`S`8HO4PLE5)`V $)STUHN](https://user-images.githubusercontent.com/3787410/184543085-b8a4e960-cc24-4ab2-9686-f5c59b9ce16b.png)